src/components/nav-bar.js

```diff
@@ -0,0 +1,40 @@
+import { html, LitElement } from "lit-element";
+import { connect } from "pwa-helpers";
+
+import { store } from "../redux/store.js";
+import { logout } from "../redux/login-actions.js";
+
+class NavBar extends connect(store)(LitElement) {
+  static get properties() {
+    return {
+      isLoggedIn: {
+        type: Object,
+      },
+    };
+  }
+
+  stateChanged(state) {
+    this.isLoggedIn = state.login.user;
+  }
+
+  logout() {
+    store.dispatch(logout());
+    window.location.href = "http://localhost:8080/";
+  }
+
+  render() {
+    const style = "font-size:32px; color:white; margin:0 20px";
+    return html`
+      <div style="background: rgb(42, 52, 67); padding: 12px 0px">
+        ${!this.isLoggedIn
+          ? html`<a href="/" style=${style}>Log In</a>
+              <a href="/sign-in" style=${style}>Sign In</a>`
+          : html` <a href="/blogs" style=${style}>Blogs</a>
+              <a href="/create" style=${style}>Create</a>
+              <a href="/" style=${style} @click=${this.logout}>Logout</a>`}
```

Comment:
Wrap prop value with quotes and use stylemap for style instead of style variable.